doc/functions/enum.rst

```diff
@@ -13,10 +13,10 @@
     {{ enum('App\\MyEnum').SomeCase.value }}
 
     {# get all cases of an enum #}
-    {% enum('App\\MyEnum').cases %}
+    {{ enum('App\\MyEnum').cases }}
```

Comment:
This does not make sense either. You cannot echo an array.

Reply:
yes I know but enum tag in {% %} doesn't work if is not in for loop it's confusing when you read the doc. 
Maybe replace by `{% for case in enum('App\\MyEnum').cases %} ` ?